R/pool.R

```diff
@@ -179,7 +179,9 @@ Pool <- R6::R6Class("Pool",
       ## detect leaked connections and destroy them
       reg.finalizer(pool_metadata, function(e) {
         if (pool_metadata$valid) {
-          warning("You have a leaked pooled object.")
+          cat("<pool> Checked-out object deleted before being returned.\n")
```

Comment:
I'm not convinced that it's even worthwhile to message here — because you on the get the message on GC, it appears a random amount of time after you've actually deleted the object, making it rather hard to pin down.